Elevel/Application/Features/QuestionCommands/GetQuestionDetailQuery.cs

```diff
@@ -32,7 +32,7 @@ public Handler(IApplicationDbContext context, IMapper mapper)
 
             public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
             {
-                var question = await _context.Questions.AsNoTracking()
+                var question = await _context.Questions.AsNoTracking().Where(x => !x.AuditionId.HasValue)
```

Comment:
it's not a bug, but it's better to keep 1 linq operator per row

Reply:
this is necessary when the line becomes unreadable IMHO